docker/build_scripts/build_utils.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+# Helper utilities for build
+
+PYTHON_DOWNLOAD_URL=https://www.python.org/ftp/python
+OPENSSL_DOWNLOAD_URL=http://www.openssl.org/source
+GET_PIP_URL=https://bootstrap.pypa.io/get-pip.py
+
+
+function check_var {
+    if [ -z "$1" ]; then
+        echo "required variable not defined"
+        exit 1
+    fi
+}
+
+
+function pyver_ge {
+    # Echo 1 if first python version is greater or equal to second
```

Comment:
Counter-proposal which generalizes to one through three digit comparison - how about:

```
lex_pyver () {
    echo $1 | awk -F "." '{printf "%03d%03d%03d\n", $1, $2, $3}'
}
```

Then:

```
if [ $(lex_pyver 3.2.1) -lt $(lex_pyver 3.3) ]; then echo "yes"; fi
```